agasc/agasc.py

```diff
@@ -132,6 +137,10 @@ def get_agasc_cone(ra, dec, radius=1.5, date=None, agasc_file=None,
     # Possibly expand initial radius to allow for slop due proper motion
     rad_pm = radius + (0.1 if pm_filter else 0.0)
 
+    global RA_DECS
```

Comment:
OK, though isn't the current API more idiomatic than suggesting that the user explicitly set agasc_file if he or she wants it to use a custom file and be fast?